yb-voyager/cmd/importData.go

```diff
@@ -242,10 +243,29 @@ func applyTableListFilter(importFileTasks []*ImportFileTask) []*ImportFileTask {
 	excludeList := utils.CsvStringToSlice(tconf.ExcludeTableList)
 	log.Infof("excludeList: %v", excludeList)
 
-	allTables := make([]string, 0, len(importFileTasks))
-	for _, task := range importFileTasks {
-		allTables = append(allTables, task.TableName)
+	//TODO: handle with case sensitivity later
+	standardizeCaseInsensitiveTableNames := func(tableName string) string {
+		parts := strings.Split(tableName, ".")
+		tableName = parts[len(parts)-1]
+		if !utils.IsQuotedString(tableName) {
+			tableName = strings.ToLower(tableName)
+		}
+		if len(parts) > 1 {
+			return fmt.Sprintf(`%s.%s`, parts[0], tableName)
```

Comment:
If the schema name is the default one i.e. the one returned from `getDefaultSourceSchemaName()`, then we should not include it in the returned name.